### PR TITLE
update and prettify README.md

### PR DESCRIPTION
**Pull request by PyDever**
I know you guys took all that time to write the docs, however the readme is the first thing a potential pallets follower and user will see. I did not edit any pre existing content, just added a badge and some installation / usage examples.

### CODE REVIEW

**Comment by bittner** [comment]
Could you also include the README in [setup.py]() using the `long_description` kwarg? This would make its content be displayed on its [PyPI project page](https://pypi.org/project/click/).

**Note** that traditionally, this would require the README to be in [reStructuredText](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) (i.e. rename it to `README.rst` and change the formatting - mostly the links and headings - slightly). I read, I think in the Python weekly newsletter a few weeks ago that it's now possible to tell PyPI the content type (or so), i.e. use MarkDown and specify that explicitly in `setup.py`.

**Comment by PyDever** [comment]
Agreed. Thanks for commenting. You might actually get this request some attention.

**Comment by davidism** [comment]
Thanks. Going forward, please use reStructuredText, not Markdown.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Hello John!
+Hello John!
+```
+
+## Installation
+The installation process is easy and contained.
+```
+$ pip install Click
+```
+Or you can build from source.
```
Comment (ThiefMaster) [inline README.md]: Is that even worth mentioning?

Comment (davidism) [reply README.md]: We're copying the README to Flask over, so this will all be normalized soon.

**Comment by bittner** [comment]
README still not included in setup.py :unamused:

**Comment by davidism** [comment]
I will update all the metadata to match the Flask repo before the next release.

**Pull request merged**